pepper-apis/src/main/antlr4/org/broadinstitute/ddp/pex/lang/Pex.g4

```diff
@@ -22,6 +22,7 @@ expr
 query
   : 'user' '.' study '.' studyPredicate                                                  # StudyQuery
   | 'user' '.' study '.' form '.' formPredicate                                          # FormQuery
+  | 'user' '.' study '.' form '.' instance '.' formInstancePredicate                     # FormInstanceQuery
```

Comment:
We need to narrow do to a particular activity instance because content substitution values are associated with an activity instance.